app/assets/stylesheets/components/new_user/_new_user.scss

```diff
@@ -0,0 +1,5 @@
+.ccs-user-form-group  {
```

Comment:
This seems un-necessary, can we look at this:

https://design-system.service.gov.uk/components/fieldset/

to see how to layout forms?